builders/README.md

```diff
@@ -14,6 +14,10 @@ any machine that can run Docker can do the cross-compilation.
 
 ## Changelog
 
+**Version 0017:**
+
+- Update Rust to 1.70.0.
```

Comment:
Can you pull out the builders update in a separate PR? We need to publish them in advance to https://hub.docker.com/r/cosmwasm/go-ext-builder/tags. This is then a standalone preparation for any potential updates. There we can also use latest stable Rust.

Reply:
sure thing